src/AppiumLibrary/__init__.py

```diff
@@ -17,6 +17,7 @@ class AppiumLibrary(
     _WaitingKeywords,
     _TouchKeywords,
     _KeyeventKeywords,
+	_AndroidUtilsKeywords,

```

Comment:
oops, sorry I set up a new dev environment and the editor was not correctly set up (use spaces instead of tabs)